### ISSUE TITLE
Errors in the Pseudospectral ebnchmarks

### ISSUE THREAD

**Issue by stmorgenstern** [SciML/SciMLBenchmarks.jl#929]
**Describe the bug 🐞**

All the pseudospectral PDE benchmarks are failing except the Burgers equation. There is no work precision diagrams generated within the docs

**Comment by ChrisRackauckas** [comment]
Yes, these have not been building for awhile and need some help.

**Comment by ChrisRackauckas** [comment]
https://github.com/JuliaApproximation/ClassicalOrthogonalPolynomials.jl/blob/main/examples/collocation.jl we should update them to use this new package
